otx/cli/tools/build.py

```diff
@@ -0,0 +1,60 @@
+"""Model templates searching tool."""
+
+# Copyright (C) 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+
+import argparse
+
+from otx.cli.builder import Builder
+
+SUPPORTED_TASKS = ("CLASSIFICATION", "DETECTION", "INSTANCE_SEGMENTATION", "SEGMENTATION")
+
+
+def parse_args():
+    """Parses command line arguments."""
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--task", help=f"The currently supported options: {SUPPORTED_TASKS}.")
+    parser.add_argument("--model", help="Input OTX model config file (e.g model.py).", default=None)
+    parser.add_argument("--backbone", help="Enter the backbone configuration file path or available backbone type.")
+    parser.add_argument("--save-backbone-to", default="./backbone.yaml")
+
+    return parser.parse_args()
+
+
+def main():
+    """Main function for model templates searching."""
```

Comment:
Same here, this description needs to be update

Reply:
fixed!